packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -12,6 +12,18 @@ model MultiPartRequest {
   profileImage: bytes;
 }
 
+model Address {
+  city: string;
+}
+
+model ComplexPartsRequest {
```

Comment:
I feel like separating those scenarios will be extra valuable too as it seems like some of those might be trickier to implement in some language

Reply:
Updated

Reply:
not sure we need this uber model and `complex` op  now too

Reply:
@timotheeguerin `complex` is strongly asked by @lmazuel. Although there is no real service to use the complex scenario, maybe it could help language owners to think about the scenario in advance.

Reply:
I don't like the name `complex` here, the name is too generic. I can imagine in future we could have **more complex** cases in once TypeSpec could allow to specify any content type for any part.